spec/system/news_article_spec.rb

```diff
@@ -1,5 +1,8 @@
 RSpec.describe "News Article" do
-  let!(:content_item) { content_store_has_example_item("/government/news/christmas-2016-prime-ministers-message", schema: :news_article) }
+  let(:news_article_christmas_2016_path) { "/government/news/christmas-2016-prime-ministers-message" }
```

Comment:
I'm not sure what the value of having this here is? I'd prefer duplication if it increases readability.

I think that trying to reduce the duplication has made the tests much harder to follow.